### ISSUE TITLE
When 2024.9.9?

### ISSUE THREAD

**Issue by vlkardakov** [scp222thj/MalumMenu#188]
...

**Comment by astra1dev** [comment]
There is no new version. MalumMenu is up-to-date with the latest Among Us version.
Please close this issue if you don't have any more questions.

**Comment by vlkardakov** [comment]
No, latest among s version is 2024.9.9, and mod latest version is 2024.9.4... https://imgur.com/a/QwVRYJI

**Comment by astra1dev** [comment]
but it works!
Theres nothing to update if it works without issues!

**Comment by scp222thj** [comment]
Is the incompatible version popup appearing when the game opens or anything?
